### PR TITLE
Send email with Template

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts [hunk 1]
```diff
@@ -61,34 +65,62 @@ export class MarketplaceMovieViewComponent implements OnInit {
...
-        message: new FormControl(),
-      })
+      data: { orgs, form }
     });
   }
 
-  sendEmail(email: EmailTemplate, movie: Movie) {
+  async sendEmail(emailData: EmailData, title: string, orgs: Organization[]) {
     this.dialogRef.close();
+    const templateId = 'd-e902521de8684c57bbfa633bad88567a';
```
Comment (phpgeekfr) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:106]: this template id cannot be stored in `/home/bruce/git/github/blockframes/apps/backend-functions/src/templates/ids.ts` ? 
Or maybe move ids.ts to lib

File: apps/financiers/financiers/src/app/dashboard/title/list/list.component.ts [hunk 2]
```diff
@@ -51,11 +48,6 @@ export class ListComponent implements OnInit, OnDestroy {
   ) { }
 
   ngOnInit() {
-    // Sync with anaytics: It's ok to give ALL movieIds they'll just be set to 0
-    this.sub = this.orgQuery.selectActive().pipe(
-      switchMap(org => this.movieService.syncWithAnalytics(org.movieIds)),
-    ).subscribe();
-
```
Comment (phpgeekfr) [inline apps/financiers/financiers/src/app/dashboard/title/list/list.component.ts:58]: Analytics not needed on this financiers page ?

**Review by phpgeekfr — APPROVED** [review]

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/marketplace/title/view/view.component.ts:106]: I put is also in the templates/ids.ts but I cannot import it from the frontend... Maybe in the campaign lib ?

File: apps/financiers/financiers/src/app/dashboard/title/list/list.component.ts (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/financiers/financiers/src/app/dashboard/title/list/list.component.ts:58]: nop

**Pull request merged**